support-models/src/main/scala/com/gu/support/workers/Products.scala

```diff
@@ -20,15 +20,13 @@ sealed trait ProductType {
 
   override def toString: String = this.getClass.getSimpleName
   def describe: String
-  val catalogType: com.gu.support.catalog.Product
 }
 
 case class Contribution(
   amount: BigDecimal,
   currency: Currency,
   billingPeriod: BillingPeriod
 ) extends ProductType {
-  override val catalogType = com.gu.support.catalog.Contribution
```

Comment:
this doesn't seem to be used, even by the serialisers.  (I know that because if the amount is missing from a contribution, it gets put through as a digital pack)